src/components/DefaultPage/DefaultPage.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from "react";
+import { connect } from "react-redux";
+import { IMovie } from "../../interfaces/IMovie";
+import { loadMovies } from "../../store/actions/loadMovies";
+import { setSearchQuery } from "../../store/actions/searchMovie";
```

Comment:
you can create an index.ts file inside the actions folder. And reexport all the actions from this file. So that you will be able to import them like that:
`import { loadMovies,  setSearchQuery} from '../../store/actions' `